src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine [] machine = new Machine [3];
+        machine[0] = new Excavator();
+        machine[1] = new Truck();
+        machine[2] = new Bulldozer();
+        for (int i = 0; i < 3; i++) {
```

Comment:
```suggestion
        Machine[] machines = new Machine[]{new Excavator(), new Truck(), new Bulldozer()};
```

we can make it simpler